.github/workflows/holonix-cache.yml

```diff
@@ -53,3 +57,33 @@ jobs:
           done
 
           rm result*
+  cache-check:
+    needs:
+      - cache-update
+    strategy:
+      fail-fast: false
+      matrix:
+        extra_args:
+          - '--override-input versions github:holochain/holochain?dir=versions/0_1'
+          - '--override-input versions github:holochain/holochain?dir=versions/0_2'
+          - '--override-input versions github:holochain/holochain?dir=versions/weekly'
+        platform:
+          - x86_64-darwin
+          - aarch64-darwin
+          - x86_64-linux
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          lfs: false
+      - uses: cachix/install-nix-action@v22
+      - uses: cachix/cachix-action@v12
+        with:
+          name: holochain-ci
+      - name: Check Holonix cache
+        uses: ThetaSinner/nix-cache-check@v1
+        with:
+          derivation: .#devShells.${{ matrix.platform }}.holonix
+          extra_build_args: ${{ matrix.extra_args }}
+          permit_build_derivations: rust-default-1.66.1,hn-introspect,nix-shell,inheritCargoArtifactsHook,cargoHelperFunctionsHook,installFromCargoBuildLogHook
```

Comment:
GitHub expects these values to be trivially [string convertible](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs) so I don't think you can use object or list types. I did try that because it would be nicer.

Another example of CSV input in an action we use https://github.com/cachix/cachix-action/blob/master/action.yml#L9